package.json

```diff
@@ -138,7 +138,7 @@
     "axios": "0.25.0",
     "es6-promise": "^4.2.4",
     "lodash": "4.17.21",
-    "stellar-base": "8.2.2-soroban.12",
+    "stellar-base": "https://github.com/stellar/js-stellar-base\\#host_functions_ts",
```

Comment:
I'll change this to `v9.0.0-soroban.1` when it's out, for now using this slightly newer stellar-base that has tweaks on txbuilder - [js-stellar-base#604](https://github.com/stellar/js-stellar-base/pull/604)

Reply:
it's out! https://www.npmjs.com/package/stellar-base/v/9.0.0-soroban.1